packages/forms/src/UIForm/Widget/condition.js

```diff
@@ -1,29 +1,173 @@
 import get from 'lodash/get';
-import includes from 'lodash/includes';
+import findIndex from 'lodash/findIndex';
 
-function getValue(value, strategy) {
-	if (!strategy) {
+/**
+ * Tries to ensure the returned value is a number.
+ * It supports string and number for now.
+ */
+function stringOrNumberToNumber(value) {
+	if (typeof value === 'number') {
 		return value;
 	}
-	switch (strategy.toLowerCase()) {
-		case 'length':
-			if (value && value.length) {
-				return value.length;
+	if (typeof value === 'string') {
+		return Number(value);
+	}
+	const error = { error: 'the passed value is not a string or a number', value };
+	throw error;
+}
+
+/**
+ * Tries to make the returned value a string.
+ * It supports number and string as input.
+ * In case of an error it will call onError. By default it throws an exception.
+ */
+function stringOrNumberToString(
+	value,
+	onError = val => {
+		const error = { error: 'the passed value is not a string or a number', value: val };
+		throw error;
+	},
+) {
+	if (!value) {
+		return value;
```

Comment:
This check can return any falsy value like 0

Reply:
agree, i picked value since it was here ;)